data/helpers.d/network

```diff
@@ -0,0 +1,9 @@
+# Check availability of a web path
+#
+# example: ynh_path_validity $domain$path
+#
+# usage: ynh_path_validity $domain_and_path
+# | arg: domain_and_path - complete path to check
+ynh_path_validity () {
```

Comment:
I think you're better than me to read this code.
https://github.com/YunoHost/yunohost/blob/unstable/src/yunohost/app.py#L888-L935

For me, it's just a control without side-effect. But, I didn't read it again.